go.mod

```diff
@@ -40,9 +40,9 @@ require (
 	github.com/google/wire v0.2.2
 	github.com/googleapis/gax-go v2.0.2+incompatible
 	github.com/lib/pq v1.1.1
-	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
 	github.com/pkg/errors v0.8.1 // indirect
+	github.com/spf13/cobra v0.0.4 // indirect
```

Comment:
These changes to go.mod and go.sum don't belong here.

Reply:
Thanks. Sigh... I'm not sure what keeps updating these things, and I have to keep actively fighting to keep these from being checked in.